examples/basic/outputs.tf

```diff
@@ -2,17 +2,27 @@
 # Outputs
 ########################################################################################################################
 
-output "cos_instance_id" {
-  description = "COS instance id"
-  value       = ibm_resource_instance.cos_instance.id
+output "watsonx_data_crn" {
+  description = "The CRN of the watsonx.data instance."
+  value       = module.watsonx_data.watsonx_data_crn
 }
 
-output "resource_group_name" {
-  description = "Resource group name"
-  value       = module.resource_group.resource_group_name
+output "watsonx_data_dashboard_url" {
+  description = "The dashboard URL of the watsonx.data."
+  value       = module.watsonx_data.watsonx_data_dashboard_url
 }
 
-output "resource_group_id" {
-  description = "Resource group ID"
-  value       = module.resource_group.resource_group_id
+output "watsonx_data_guid" {
+  description = "The GUID of the watsonx.data instance."
+  value       = module.watsonx_data.watsonx_data_guid
+}
+
+output "watsonx_data_name" {
+  description = "The name of the watsonx.data instance."
+  value       = module.watsonx_data.watsonx_data_name
+}
+
+output "watsonx_data_plan_id" {
```

Comment:
since the module creates a resource group if one is not provided we should output the resource group name and id

Reply:
you mean the example creates the resource group, not the module